.github/workflows/optimize-images.yml

```diff
@@ -0,0 +1,61 @@
+# Release builds that are triggered by the v*.*.* tag. Release tags occur when
+# someone comments /approve on the associated release ticket
+
+name: '[Build] Release Build'
```

Comment:
Looks like these were copied over from the release workflow. Please rename workflow and update comment above describing the workflow.

Reply:
good catch